spaceR/spaceR/Models/Launches.swift

```diff
@@ -1,64 +1,109 @@
 //
 //  Launches.swift
-//  spaceR
+//  SpaceR
 //
 //  Created by Kieran Woodrow on 2022/02/24.
 //
 
 import Foundation
 
-struct Launch: Codable{
+struct Launch: Codable {
     let fairlings: Fairlings?
     let links: Links?
-    let static_fire_date_utc, details, name, date_utc, date_local: String?
+    let staticFireDateUtc, details, name, dateUtc, dateLocal: String?
     let success, upcoming: Bool?
     let failures: [Failures?]
     let crew: [Crew?]
     let ships, capsules: [UUID?]
-    let flight_number: Int?
+    let flightNumber: Int?
     let cores: [Cores?]
     let id, rocket, launchpad: UUID?
+    private enum CodingKeys: String, CodingKey {
```

Comment:
Much better, great job 👍

Reply:
Maybe also just add a line between 20 and 21 😄

Reply:
perfect